### ISSUE TITLE
Formalize and implement another option in widgets, the defaultValue

### ISSUE THREAD

**Issue by tiberiuichim** [plone/volto#1799]
Now that we've started using client-side forms based on schemas, I notice that we don't have a "defaultValue" property for the fields that can be respected by the schema form implementation.

**Comment by nileshgulia1** [comment]
defaultValue is for the first load right? Since we're handling Form state in `Form.jsx` we need to think of building the controlled component.  I like to just use defaultValue if I'm just initializing it and then just use refs to get the value when I submit.

**Comment by tiberiuichim** [comment]
I don't think it should concern the form what happens with the default value. The form should always pass the defaultValue if it exists in the schema and the widgets implementation should do the right thing: use the defaultValue as the initial value, but don't set it in the form state except when the widget triggers onChangeField. I think it would be a complication to try to write that defaultValue in the initial form state.
